vaadin-app-layout-flow-parent/vaadin-app-layout-flow-integration-tests/vite.config.ts

```diff
@@ -0,0 +1,27 @@
+// @ts-ignore can not be resolved until NPM packages are installed
+import { UserConfigFn } from 'vite';
+// @ts-ignore can not be resolved until Flow generates base Vite config
+import { overrideVaadinConfig } from './vite.generated';
+// import { useLocalWebComponents } from '../../shared/web-components-vite-plugin';
+
+const customConfig: UserConfigFn = (env) => ({
+  // Here you can add custom Vite parameters
+  // https://vitejs.dev/config/
+
+  // Disable the Vite deps pre-bundling in the app to prevent Vite
+  // from caching connectors’ source code in the file system
+  optimizeDeps: {
+    disabled: true
+  },
+
+  // Use local version of web-components, disabled by default
+  // To use this un-comment the lines below and adapt the path to your local checkout
```

Comment:
Let's maybe mention that it should be an absolute path to `node_modules`, because it can be not immediately noticeable from the path below.

Reply:
> NOTE: Make sure that the path is an absolute one and that it points to the node_modules directory in the web components monorepo.

Like we have in the web components README.

Reply:
Done